### PR TITLE
Invert default visibility

### PR DESCRIPTION
**Pull request by klingbolt**
Changed Default_toggle to hide_category in the proto so the default can be 0. #151

### CODE REVIEW

**Comment by klingbolt** [comment]
I am still uncertain if hide_category or something like hide_by_default. I felt like adding the word default is unnecessary but on the other hand, the current behavior of Burrito is to hide all categories by default so we may want to change this behavior in the future.

File: xml_converter/proto/waypoint.proto [hunk 1]
```diff
@@ -21,7 +21,7 @@ message Category {
     repeated Icon icon = 3;
     repeated Trail trail = 4;
     bool is_separator = 5;
-    bool default_visibility = 6;
+    bool hide_category = 6;
```
Comment (AsherGlick) [inline xml_converter/proto/waypoint.proto:24]: What were other names that were considered here and why did you decide on `hide_category`?
As it is written now this field is `Category.hide_category` which seems redundant.
Just removing `category` and making it `Category.hide` might look like a good idea initially but will it make intuitive sense when combined with the hidden/shown safe file data too?

Comment (klingbolt) [reply xml_converter/proto/waypoint.proto:24]: The other top names I considered were `hide_by_default' and `default_hide_category`.
There is another field in the XML called `hide` that we made `trigger.action_hide_category` so I used that naming convention. Though on further thought, that field is a Category object and this one is a bool so maybe they don't need to be similar. 
I considered using `hide` but was concerned that it was too similar to the Godot function `hide()` and that when accessing this field, `Category.get_hide()` isn't descriptive enough.

Comment (AsherGlick) [reply xml_converter/proto/waypoint.proto:24]: My gut tells me that `hide_category` is too imperative / verb-ish when it should be more declarative / noun-ish.

Comment (klingbolt) [reply xml_converter/proto/waypoint.proto:24]: I agree

How about `is_hidden`? This makes it clear its a property not a command. Also fits with some other attributes like `is_separator` and `is_wall`.

**Pull request merged**